include/SDL3/SDL_main.h

```diff
@@ -95,6 +95,7 @@
         #define SDLMAIN_DECLSPEC    SDL_DECLSPEC
 
     #elif defined(SDL_PLATFORM_EMSCRIPTEN)
+#if 0 /* FIXME: disabled for stdout/stderr */
```

Comment:
@icculus 
This undoes a change in https://github.com/libsdl-org/SDL/pull/10171#discussion_r1720765567, and is not a fix.

Doing `SDL_MAIN_HANDLED` in the tests (for emscripten only) would also work, but that is also not a fix.